manifests/config.pp

```diff
@@ -18,6 +18,8 @@
   $cvmfs_repo_list                     = $cvmfs::cvmfs_repo_list,
   $cvmfs_memcache_size                 = $cvmfs::cvmfs_memcache_size,
   $cvmfs_claim_ownership               = $cvmfs::cvmfs_claim_ownership,
+  $cvmfs_uid_map                       = $cvmfs::cvmfs_uid_map,
```

Comment:
Please can you update the README with the new parameters.

Reply:
Done, let me know if the level of detail is fine.